docs/book/tutorials/kubernetes-on-vsphere-with-kubeadm.md

```diff
@@ -658,76 +658,10 @@ If you are not interested in K8s object placement, this section can be ignored a
 
 ### Create Roles, ServiceAccount and ClusterRoleBinding
 
-In these steps, the ClusterRole, ServiceAccounts and ClusterRoleBinding needed for installation of vSphere CSI Driver are created. All of these are in the following manifest. Copy and paste it into your environment. No modifications are required. Take care when copying/pasting as the manifest files are rather large.
+In these steps, the ClusterRole, ServiceAccounts and ClusterRoleBinding needed for installation of vSphere CSI Driver are created. All of these are in the following manifest. Simply use `kubectl` to import the manifest into your cluster.
```

Comment:
Prefer not to use terms like "Simply" in docs cause it implies something is "easy" or "obvious" but when it may not be for everyone. 

Thoughts on
```
You can use `kubectl` to import the manifest into your cluster.
```
?